### PR TITLE
Clear doctest warnings

### PR DESCRIPTION
**Pull request by niboshi**
See: #442

### CODE REVIEW

File: cupy/random/__init__.py [hunk 1]
```diff
@@ -1,4 +1,13 @@
-from numpy.random import bytes   # NOQA
+import numpy
+
+
+def bytes(length):
+    """Return random bytes.
```
Comment (okuta) [inline cupy/random/__init__.py]: `Returns`

**Comment by niboshi** [comment]
Fixed.

**Comment by okuta** [comment]
Cloud you rebase this PR

**Comment by niboshi** [comment]
Rebased.

**Comment by okuta** [comment]
jenkins, test this please

**Comment by okuta** [comment]
jenkins, test this please

**Pull request merged**